wiki/Gameplay/Tablet_cover/de.md

```diff
@@ -0,0 +1,42 @@
+---
+stub: true
+tags:
+  - tablet
+  - scratch
+  - nibwear
+  - tablet cover
+  - tablet abdeckung
+  - schutzfolien
+  - kratzer
+---
+
+<!-- TODO:
+- add images for DIY and store-bought tablet covers
+- include information on the materials used for manufactured tablet covers -->
+
+# Tablet-Abdeckung
+
+<!--See also: [Tablet](link)-->
+
+Eine **Tablet-Abdeckung** oder **Tablet-Cover** ist eine Folie aus speziellem Schutzmaterial, die die Oberfläche eines Grafiktabletts umhüllt oder aufklebt und so zusätzlichen Schutz bietet. Diese Abdeckungen ähneln Smartphone-[Displayschutzfolien](https://de.wikipedia.org/wiki/Displayschutzfolie), wobei die einzigen wesentlichen Unterschiede in Form und Design bestehen. Und obwohl Tabletten-Abdeckungen letztlich denselben Zweck erfüllen wie die meisten anderen Displayschutzfolien, sind Tabletten-Abdeckungen oft speziell darauf ausgelegt, den Verschleiß von Stiftspitzen zu verringern, die Lebensdauer des Tablets zu verlängern und die Oberfläche des Tablets vor Kratzern zu schützen. Einige Tablet-Cover, vor allem solche, die speziell für osu! designt wurden, können auch eine Oberfläche erzeugen, die glänzender, glatter oder rauer ist, je nach Art und Vorlieben des Nutzers.
+
+Tablet-Abdeckungen werden oft in einer Form angeboten, die fast identisch mit Smartphone-Bildschirmschutzfolien ist, mit der Ausnahme, dass auf der Seite, die klebt, nur ein Streifen klebrigen Materials entlang der äußeren Kanten des Schutzmaterials verwendet wird. Dadurch wird eine Beschädigung des Materials der Tablettoberfläche vermieden und die Verwendung der Druckempfindlichkeit und anderer Funktionen für das digitale Zeichnen ermöglicht.
+
+## DIY vs. hergestellt
+
+In vielen Forenbeiträgen, in denen gefragt wird, wo man Tablet-Cover kaufen oder herstellen kann, wird oft erwähnt, wie einfach es ist, ein solches Cover zu erstellen. Für gängige selbstgemachte Tablet-Cover werden verschiedene Materialien verwendet, die zu Hause oder in Geschäften relativ günstig zu bekommen sind. Bei der Herstellung von Tablet-Abdeckungen werden hingegen häufig spezielle Materialien verwendet, um den Verschleiß von Stiftspitzen zu verringern und die Lebensdauer zu verlängern.
+
+### DIY
+
+Viele Spieler werden oft behaupten, dass DIY-Tablet-Cover "den Zweck erfüllen" und daher aufgrund ihrer geringen Kosten und ihrer Zugänglichkeit vorzuziehen sind. Für selbstgemachte Abdeckungen werden oft Materialien verwendet, die in den meisten Haushalten zu finden sind, und sie werden oft mit Klebeband oder einem schweren Gegenstand befestigt.
+
+Übliche DIY-Tablet-Abdeckungen bestehen häufig aus einem oder mehreren der folgenden Materialien:
+
+- Buchschutzfolie
+- Laminiertes Papier
+- Bill strips
```

Comment:
i don't know what bill strips are and also couldn't find anything about it online. lol
if anyone knows what those are please tell me, otherwise i will probably just leave it as the english term 🤔

Reply:
my best effort is that's a misspelling of [bill straps](https://en.wikipedia.org/wiki/Currency_strap)

Reply:
i guess that could be it, but even then it don't see how anyone would use those as a tablet cover.

Reply:
yeah, it's a misspelling of bill strap. that line prob should've been removed before merge, but yeah i remember hearing about some guy on the forums using bill straps as a diy tablet cover

Reply:
oh that's interesting. should i open another PR removing it from the other languages or just keep it?